velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp

```diff
@@ -113,6 +113,7 @@ class ArrowBridgeArrayExportTest : public testing::Test {
       const ArrowArray& arrowArray) {
     const bool isString =
         std::is_same_v<T, StringView> or std::is_same_v<T, std::string>;
+    const bool isNullType = std::is_same_v<T, UnknownValue>;
```

Comment:
nit: In Spark, the mapped type is NullType, whereas in Presto, it is UnknownType. For consistency, Velox can still refer to it as unknown type.

https://github.com/prestodb/presto/blob/master/presto-common/src/main/java/com/facebook/presto/common/type/UnknownType.java